modules/nw-ptp-configuring-linuxptp-services-as-grandmaster-clock.adoc

```diff
@@ -1,26 +1,35 @@
 // Module included in the following assemblies:
 //
+// * networking/ptp/configuring-ptp.adoc
 // * networking/using-ptp.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="configuring-linuxptp-services-as-grandmaster-clock_{context}"]
 = Configuring linuxptp services as a grandmaster clock
 
-You can configure the `linuxptp` services (`ptp4l`, `phc2sys`, `ts2phc`) as grandmaster clock by creating a `PtpConfig` custom resource (CR) that configures the host NIC.
+You can configure the `linuxptp` services (`ptp4l`, `phc2sys`, `ts2phc`) as grandmaster clock (T-GM) by creating a `PtpConfig` custom resource (CR) that configures the host NIC.
 
 The `ts2phc` utility allows you to synchronize the system clock with the PTP grandmaster clock so that the node can stream precision clock signal to downstream PTP ordinary clocks and boundary clocks.
 
 [NOTE]
 ====
-Use the following example `PtpConfig` CR as the basis to configure `linuxptp` services as the grandmaster clock for your particular hardware and environment.
-This example CR does not configure PTP fast events. To configure PTP fast events, set appropriate values for `ptp4lOpts`, `ptp4lConf`, and `ptpClockThreshold`.
+Use the following example `PtpConfig` CR as the basis to configure `linuxptp` services as T-GM for an Intel Westport Channel E810-XXVDA4T network interface.
+
+To configure PTP fast events, set appropriate values for `ptp4lOpts`, `ptp4lConf`, and `ptpClockThreshold`.
 `ptpClockThreshold` is used only when events are enabled.
 See "Configuring the PTP fast event notifications publisher" for more information.
 ====
 
 .Prerequisites
 
-* Install an Intel Westport Channel network interface in the bare-metal cluster host.
+* Install an Intel E810 Westport Channel network interface in the bare-metal cluster host.
++
+[IMPORTANT]
+====
+The Intel Westport Channel E810-XXVDA4T NIC in-tree ice driver version must be 5.14.0-356.bz2232515.el9.x86_64 or later.
```

Comment:
This might be obsolete. The driver is shipped with os

Reply:
Removing